.github/labeler.yml

```diff
@@ -0,0 +1,211 @@
+  - LICENSE
+
+installation:
+  - pyproject.toml
+  - 'setup.*'
+  - 'requirements-*.txt'
+  - MANIFEST.in
+
+ami:
+  - 'jwst/ami/**/*'
+
+assign_mtwcs:
+  - 'jwst/assign_mtwcs/**/*'
+
+assign_wcs:
+  - 'jwst/assign_wcs/**/*'
+
+associations:
+  - 'jwst/associations/**/*'
+
+background:
+  - 'jwst/background/**/*'
+
+barshadow:
+  - 'jwst/barshadow/**/*'
+
+combine_1d:
+  - 'jwst/combine_1d/**/*'
+
+coron3 pipeline:
+  - 'jwst/coron/**/*'
+  - jwst/pipeline/calwebb_coron.py
+
+cube_build:
+  - 'jwst/cube_build/**/*'
+
+cube_skymatch:
+  - 'jwst/cube_skymatch/**/*'
+
+dark_current:
+  - 'jwst/dark_current/**/*'
+
+datamodels:
+  - 'jwst/datamodels/**/*'
+
+dq_init:
+  - 'jwst/dq_init/**/*'
+
+engdb_tools:
+  - 'jwst/lib/engdb_*'
+
+exp_to_source:
+  - 'jwst/exp_to_source/**/*'
+
+extract_1d:
+  - 'jwst/extract_1d/**/*'
+
+extract_2d:
+  - 'jwst/extract_2d/**/*'
+
+firstframe:
+  - 'jwst/firstframe/**/*'
+
+fits_generator:
+  - 'jwst/fits_generator/**/*'
+
+flatfield:
+  - 'jwst/flatfield/**/*'
+
+fringe:
+  - 'jwst/fringe/**/*'
+
+gain_scale:
+  - 'jwst/gain_scale/**/*'
+
+group_scale:
+  - 'jwst/group_scale/**/*'
+
+guider_cds:
+  - 'jwst/guider_cds/**/*'
+
+imprint:
+  - 'jwst/imprint/**/*'
+
+ipc:
+  - 'jwst/ipc/**/*'
+
+jump:
+  - 'jwst/jump/**/*'
+
+lastframe:
+  - 'jwst/lastframe/**/*'
+
+linearity:
+  - 'jwst/linearity/**/*'
+
+master_background:
+  - 'jwst/master_background/**/*'
+
+mrs_imatch:
+  - 'jwst/mrs_imatch/**/*'
+
+msa_flagging:
+  - 'jwst/msaflagopen/**/*'
+
+outlier_detection:
+  - 'jwst/outlier_detection/**/*'
+
+path loss:
+  - 'jwst/pathloss/**/*'
+
+persistence:
+  - 'jwst/persistence/**/*'
+
+photom:
+  - 'jwst/photom/**/*'
+
+pipeline:
+  - 'jwst/pipeline/**/*'
+
+ramp_fitting:
+  - 'jwst/ramp_fitting/**/*'
+
+refpix:
+  - 'jwst/refpix/**/*'
+
+regression_testing:
+  - 'jwst/regtest/**/*'
+  - 'Jenkinsfile*'
+
+resample:
+  - 'jwst/resample/**/*'
+
+reset:
+  - 'jwst/reset/**/*'
+
+residual_fringe:
+  - 'jwst/residual_fringe/**/*'
+
+rscd:
+  - 'jwst/rscd/**/*'
+
+saturation:
+  - 'jwst/saturation/**/*'
+
+skymatch:
+  - 'jwst/skymatch/**/*'
+
+source catalog:
+  - 'jwst/source_catalog/**/*'
+
+source type:
+  - 'jwst/srctype/**/*'
+
+stpipe:
+  - 'jwst/stpipe/**/*'
+
+straylight:
+  - 'jwst/straylight/**/*'
+
+superbias:
+  - 'jwst/superbias/**/*'
+
+testing:
+  - '**/tests/**/*'
+  - .github/workflows/*.yml
+  - .bandit.yaml
+  - .codecov.yml
+  - conftest.py
+  - tox.ini
+
+timeconversion:
+  - 'jwst/timeconversion/**/*'
+
+transforms:
+  - 'jwst/transforms/**/*'
+
+tso_photometry:
+  - 'jwst/tso_photometry/**/*'
+
+tweakreg:
+  - 'jwst/tweakreg/**/*'
+
+wavecorr:
+  - 'jwst/wavecorr/**/*'
+
+wfs_combine:
+  - 'jwst/wfs_combine/**/*'
+
+wfss_contam:
+  - 'jwst/wfss_contam/**/*'
+
+white_light:
+  - 'jwst/white_light/**/*'
+
+Wide Field Slitless Spec (WFSS):
+  - 'jwst/**/*wfss*/**/*'
+
+NIRSPEC:
+  - 'jwst/**/*nirspec*/**/*'
+
+NIRCAM:
+  - 'jwst/**/*nircam*/**/*'
+
+NIRISS:
+  - 'jwst/**/*niriss*/**/*'
```

Comment:
I'm not sure if each instruments has specific files that are exclusive to them, so I used general patterns here.

Reply:
It's rare, but there are a few instances here and there of modules that have an instrument-specific name.